auditorium/views/decorators/with-layout.js

```diff
@@ -8,14 +8,37 @@ function withLayout (headline) {
       var flash = state.flash
       state.flash = null
       return html`
-        <div data-role="app-host" class="mw8 center pa3 pb5 f5 roboto dark-gray">
-          <div id="headline">
-            <h1 class="f2 normal mt0 mb4">${headline || __('Offen Auditorium')}</h1>
-            ${flash ? html`
-              <p data-role="flash-message" class="dib pa2 br2 bg-black-05 mt0 mb2">${flash}</p>
-            ` : null}
+        <div data-role="app-host" class="f5 roboto dark-gray">
+          <div class="w-100 h3 bg-black-05">
+            <div class="mw8 center flex ph3 pt2" id="headline">
+              <img src="/offen-icon-black.svg" alt="Offen logo" width="37" height="40" class="ma0 mt1 mr3">
+              <h1 class="f2 normal ma0 mt1">${headline || __('Offen Auditorium')}</h1>
+            </div>
+          </div>
+          <div class="mw8 center ph0 ph3-ns pb4">
+            <div>
```

Comment:
I can remove the wrapping `div` just fine and get the same result. Do we really need it?

Reply:
See the comment about working around the `nanocomponent` issue above.

Reply:
It fixed for me the `nanocomponent` issue - if we can solve it otherwise i'm happy to get rid of it